allchblk.c

```diff
@@ -1005,6 +1005,12 @@ STATIC struct hblk *GC_allochblk_nth(size_t lb_adjusted, int k,
     return hbp;
 }
 
+# if defined(VALGRIND_TRACKING)
```

Comment:
Add comment: This function is called from the sweeper whenever an object is freed.

Reply:
Fixed